backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5754990_sys_FindBackupTablesAndLogThem.sql

```diff
@@ -0,0 +1,53 @@
+-- Searching for all backup tables and insert records in backup.backup_tables 
+DO
+$$
+    DECLARE
+        r record;
+        v_created_at timestamptz;
+        v_source_table_name text;
+    BEGIN
+        FOR r IN
+            SELECT tablename
+            FROM pg_tables
+            WHERE schemaname = 'backup'
+              AND tablename LIKE '%_bkp_%'
+            LOOP
+                BEGIN
+                    -- Extract timestamp from table name
+                    BEGIN
+                        v_created_at := to_timestamp(
+                                substring(r.tablename FROM '_bkp_(\d{8}_\d{6})'),
+                                'YYYYMMDD_HH24MISS'
+                                        );
+                    EXCEPTION
+                        WHEN OTHERS THEN
+                            RAISE WARNING 'Could not parse timestamp from %', r.tablename;
+                            CONTINUE;
```

Comment:
yes raise warning,
but consider v_created_at := now()

aim: collect all `*BKP*` tables and do best effort to get more info, but if that's not possible and assume some defaults.